scripts/reana_bench.py

```diff
@@ -358,8 +473,8 @@ def _build_submitted_results_path(workflow: str) -> Path:
     return Path(f"{workflow}_submitted_results.csv")
```

Comment:
I still see `original_results.csv` when I run the script locally?

```console
$ ls -1 roofit5yadage_*
roofit5yadage_analyzed_results.csv
roofit5yadage_execution_progress_1_5.png
roofit5yadage_execution_status_1_5.png
roofit5yadage_histogram_pending_time_1_5.png
roofit5yadage_histogram_runtime_1_5.png
roofit5yadage_histogram_total_time_1_5.png
roofit5yadage_original_results.csv
roofit5yadage_submitted_results.csv
```

Reply:
Yep. It should be like that. `original results` are merged from submitted results (when we asked REANA to start workflows) and collect results. Naming is still bad. `original_results` is an output of collect so maybe `collect_results` would be a better name. WDYT?

Reply:
`collected_results` then... Using  past tense after the action.  (Like for submitted, analyzed.)

Reply:
I could also rename `submitted_results` to `started_results`. WDYT?

Reply:
P.S *warning*, you will need to rename already existing results to `collected_results`.

Reply:
We might need quite some renaming of functions and variables in the code as well :sweat: 

The logic is submit=create+upload -> submitted, start -> started, collect -> collected, analyse -> analysed. (using British English -ise over American English -ize)

But we could do it after merge as well, the PR works great :+1: